crates/pallet-feeds/src/feed_processor.rs

```diff
@@ -40,23 +105,26 @@ pub trait FeedProcessor<FeedId> {
     }
 
     /// puts a feed and returns the Metadata if any
-    /// this is called once per extrinsic that puts a feed into a give feed.
+    /// this is called once per extrinsic that puts a feed into a given feed stream.
```

Comment:
General theme: can we write comments as sentences starting with capital letter (I tend to omit dot if there is one sentence in the comment, but don't have strong preference otherwise)?:
```
    /// Puts a feed and returns the Metadata if any.
    /// This is called once per extrinsic that puts a feed into a given feed stream.
```

It helps with readability especially when like here there are 2 sentences.